src/ripple/app/main/Application.cpp

```diff
@@ -844,7 +854,9 @@ class ApplicationImp
 
         mValidations->flush ();
 
-        m_overlay->saveValidatorKeyManifests (getWalletDB ());
+        validators_->stop ();
+
+        manifestCache_->save (getWalletDB ());
```

Comment:
@wilsonianb yes, that's fine. At any rate, it appears @vinniefalco has concerns about this suggestion. Vinnie, do you prefer we keep `wallet.db` around?